project/HaHaFi/proposal_2/README.md

```diff
@@ -0,0 +1,64 @@
+# Auto video caption and download 
+## Introdution
+This is a tool which want to download the video on the web page, and get the English caption of the video without using any speech-to-text api. Just use your local GPU.
+
+The tool has two part.
+1. Get the m3u8 url in web page and download the video (in C++).
```

Comment:
Downloading from internet is related to an I/O system.  It is important, but has nothing to do with numerical calculation.